xikolo-ios.xcodeproj/project.pbxproj

```diff
@@ -1128,6 +1134,14 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		020E853E2449E103006BD700 /* Scenes */ = {
```

Comment:
Now now there is no need for an additional folder/group. Just place `CourseSceneDelegate.swift` next to `SceneDelegate.swift`